packages/ui/src/Button.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import * as React from 'react';
+
+/**
+ * The class name added to all UI components.
+ */
+const UI_CLASS = 'jp-ui';
+
+/**
+ * The class name added to buttons.
+ */
+const BUTTON_CLASS = 'jp-button';
+
+/**
+ * The class name added to styled buttons.
+ */
+const STYLED_CLASS = 'jp-mod-styled';
+
+
+/**
+ * The properties of the Button component.
+ */
+export
+interface IButtonProps {
+  children?: any;
+  className?: string;
+  onClick?: (event?: React.MouseEvent<HTMLElement>) => void;
+  tooltip?: string;
+  [key: string]: any;
+}
+
+
+/**
+ * A base button comopnent.
+ */
+export
+class Button extends React.Component<IButtonProps> {
```

Comment:
Since this seems stateless, what do you think about writing it as a functional react component?